build.sbt

```diff
@@ -24,6 +24,9 @@ lazy val microservice = Project(appName, file("."))
     scalacOptions ++= Seq(
       "-Wconf:src=routes/.*:s", // Suppress warnings in route files
       "-Wconf:msg=parameter.*is never used:s", // Suppress unused parameter warnings
+      "-Wconf:msg=Flag.*set repeatedly:s",
+      "-Wconf:msg=Setting -Wunused set to all redundantly:s",
+      "-Wconf:msg=Unreachable case except for null.*:s",
```

Comment:
Would you mind doing something like the changes to `build.sbt` from https://github.com/hmrc/pillar2-stubs/pull/100/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91? Ideally we wouldn't suppress the warnings and fix at source. That way we shouldn't need any additional compiler settings 🤞